schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -976,6 +976,12 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.json#/resourceDefinitions/virtualMachineScaleSets" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachines_extensions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachineScaleSets_extensions" },
```

Comment:
Please also add to `schemas/2019-04-01/deploymentTemplate.json` as per the repo readme.